src/server/es/aggs.js

```diff
@@ -536,6 +536,7 @@ export const textAggregation = async (
     defaultAuthFilter,
     nestedAggFields,
     nestedPath,
+    numericField,
```

Comment:
(Optional) What do you think of renaming this and the other instances of `numericField` to `isNumericField`? I didn't initially understand what `numericField` meant, but as I understand it it's a boolean flag saying whether the field we're aggregating over is numeric (?)

Reply:
oh yeah that would make it more clear!